Source/WebKit/UIProcess/Network/NetworkProcessProxy.messages.in

```diff
@@ -111,6 +111,8 @@ messages -> NetworkProcessProxy LegacyReceiver {
 #if ENABLE(NETWORK_ISSUE_REPORTING)
     ReportNetworkIssue(WebKit::WebPageProxyIdentifier webPageProxyIdentifier, URL requestURL)
 #endif
+
+    AddAllowedFirstPartyForCookies(WebCore::ProcessIdentifier webProcessIdentifier, WebCore::RegistrableDomain firstPartyDomain)
```

Comment:
This should probably go through the NetworkConnectionToWebProcess instead of through the UI process.